ocfweb/stats/semester_job.py

```diff
@@ -0,0 +1,109 @@
+from datetime import date
+from datetime import datetime
+from datetime import timedelta
+
+import ocflib.printing.quota as quota
+from django.http import HttpResponse
+from matplotlib.figure import Figure
+from matplotlib.ticker import MaxNLocator
+from ocflib.printing.quota import WEEKDAY_QUOTA
+from ocflib.printing.quota import WEEKEND_QUOTA
+
+from ocfweb.component.graph import canonical_graph
+from ocfweb.component.graph import plot_to_image_bytes
+
+ONE_DAY = timedelta(days=1)
+MONTH_INDEX = 1
+SEMESTER_CUTOFF_MONTH = 8
+
+year = datetime.today().year
+spring_start = date(year, 1, 1)
+fall_start = date(year, 8, 1)
+
+
+def get_start_end():
+    """reutnr a tuple of startdate and endate of the semester based on today"""
+    day = datetime.today().timetuple()
+    if day[MONTH_INDEX] < SEMESTER_CUTOFF_MONTH:
```

Comment:
`MONTH_INDEX` makes no sense as a constant; i had no idea what it meant when i saw it at the top of the file

instead you could either just:
1. use `datetime.today().month`
2. use `datetime.today().timetuple().tm_mon`?